src/api/Main/DataLayer/Shrooms.DataLayer/Migrations/201904131312193_AddKudosWelcomeColumns.cs

```diff
@@ -0,0 +1,22 @@
+namespace Shrooms.DataLayer.Migrations
+{
+    using System;
+    using System.Data.Entity.Migrations;
+    
+    public partial class AddKudosWelcomeColumns : DbMigration
+    {
+        public override void Up()
+        {
+            AddColumn("dbo.Organizations", "KudosWelcomeAmount", c => c.Int(nullable: false));
```

Comment:
Is this migration working well on existing DB, having entries in Organizations table?
It looks like new added not nullable column might have issues with existing data.